### PR TITLE
extended Support to ServerHw_ServerHwTypes_LIG_LI_Interconnect_InterconnectType_NetworkSet

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Extended Support to ServerHw_ServerHwTypes_LIG_LI_Interconnect_InterconnectType_NetworkSet

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/logical_interconnects.go [hunk 1]
```diff
@@ -19,11 +19,11 @@ func newFalse() *bool {
...
 	var (
 		clientOV        *ov.OVClient
-		id              = "7a8b37a6-bde0-4d06-8d20-6e574c50f39f"
+		id              = "e6f42b51-ba5a-430e-8b68-a9d1de223293"
 		macAddress      = "94:57:A5:67:2C:BE"
-		internalVlan    = "504"
-		interconnectURI = "/rest/interconnects/b6b7325f-666f-474f-a8f7-2c32b3c9faab"
-		externalVlan    = "504"
+		internalVlan    = "1052"
+		interconnectURI = "rest/ethernet-networks/577d8021-67e1-4d3e-92a1-fa20f61c812a"
```
Comment (VenkateshRavula) [inline examples/logical_interconnects.go:25]: Why did this uri changed from interconnects to ethernet-networks..? Any reason

File: examples/interconnect_type.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ func main() {
 		fmt.Println(interconnect)
 	}
 
-	interconnect_type := utils.NewNstring("/rest/interconnect-types/a3af4d5e-7114-4e7a-a6c4-f97b707ec87c")
+	interconnect_type := utils.NewNstring("/rest/interconnects/e1242fc7-4e54-488d-90b2-2f8469a2b80a")
```
Comment (VenkateshRavula) [inline examples/interconnect_type.go:32]: why interconnect-types changed to interconnects..?

Comment (AsisBagga) [reply examples/interconnect_type.go]: i was testing something, forgot to put it back to defaults. I have fixed it, let me know

File: examples/logical_interconnects.go (hunk 1 above)
Comment (AsisBagga) [reply examples/logical_interconnects.go]: i was testing something, forgot to put it back to defaults. I have fixed it, let me know

**Pull request merged**